File: app/src/main/java/com/neome/api/meta/base/dto/StudioPluginJar.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class StudioPluginJar extends StudioBase
{
  public Date lastUpdateTime;

  @Nullable
  public MetaIdVar packageNameVarId;

  @Nullable
  public MetaIdForm pluginConfigFormId;

  @Nullable
  public StudioMapOfJarFile uploadJarMap;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioPluginJar.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioMapOfJarFile

interface StudioPluginJar : StudioBase
{
  val lastUpdateTime: String
  val packageNameVarId: MetaIdVar?
  val pluginConfigFormId: MetaIdForm?
  val uploadJarMap: StudioMapOfJarFile?
}